src/components/RichTextEditor/RichTextEditor.css

```diff
@@ -33,12 +33,11 @@
 }
 
 .menu-bar-container-hidden {
-  height: 0;
+  height: var(--menu-bar-height-hidden, auto);
```

Comment:
Could you please keep the default value the same as before you add this variable?